examples/variant-prediction/predict.py

```diff
@@ -156,7 +156,7 @@ def main(args):
             model = model.cuda()
             print("Transferred model to GPU")
 
-        batch_converter = alphabet.get_batch_converter()
+        batch_converter = alphabet.get_batch_converter(args.truncate)
```

Comment:
This wasn't added to `parser`. But tbh I'd leave this script as-is; if you pass in a `--sequence` longer than allowed you'll run into all kinds of trouble like pseudo-perplexity (`compute_pppl`) is undefined, and if you have mutations beyond max_length truncation means they be just ignored